src/pages/docs/ai-transport/features/sessions-identity/online-status.mdx

```diff
@@ -0,0 +1,279 @@
+
+**Multi-device coordination**: A single user can connect from multiple devices simultaneously. Presence tracks each connection separately while maintaining stable identity across devices, allowing you to distinguish between "one device left" and "user completely offline".
+
+**Agent availability signaling**: Clients need to know when agents are online and ready to process requests. Agents can enter presence to advertise availability and leave when they complete work or shut down.
+
+**Collaborative session awareness**: In sessions with multiple users, participants can see who else is currently present. This enables realtime collaboration features and helps users understand the current session context.
+
+## Going online <a id="going-online"/>
+
+Use the [`enter()`](/docs/presence-occupancy/presence#enter) method to signal that a user or agent is online. You can optionally include data when entering presence to communicate additional context.
+
+You have flexibility in when to enter presence. For example, an agent might choose to appear as online only while processing a specific task, or remain present for the duration of the entire session. Users typically enter presence when they connect to a session and remain present until they disconnect.
+
+<Aside data-type="note">
+Clients require the [`presence`](/docs/auth/capabilities) capability on the channel to enter presence.
+</Aside>
+
+<Code>
+```javascript
+// Client code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Enter presence with metadata about the user's device
+await channel.presence.enter({
+  device: "mobile",
+  platform: "ios"
+});
+```
+</Code>
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}",
+  clientId: "weather-agent"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Enter presence with metadata about the agent
+await channel.presence.enter({
+  model: "gpt-4"
+});
+```
+</Code>
+
+### Going online from multiple devices <a id="multiple-devices"/>
+
+A single user can be present on a channel from multiple devices simultaneously. Ably tracks each connection separately using a unique [`connectionId`](/docs/connect#connection-ids), while maintaining the same [`clientId`](/docs/auth/identified-clients#assign) across all connections.
+
+When a user connects from multiple devices, each device enters presence independently. All connections share the same `clientId` but have different `connectionId` values:
+
+<Code>
+```javascript
+// Client code (device 1: desktop browser)
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+await channel.presence.enter({ device: "desktop" });
+```
+</Code>
+
+<Code>
```

Comment:
I feel we should have some text (not sure what) in between two code blocks.

Maybe, the comment showing device 1 and device 2 could be text above each? Breaking up the read from one code block immediately into another.